aspnetcore/fundamentals/w3c-logger/index.md

```diff
@@ -0,0 +1,115 @@
+---
+title: W3CLogger in .NET Core and ASP.NET Core
+author: wtgodbe
+description: Learn how to create server logs in the W3C standard format.
+monikerRange: '>= aspnetcore-6.0'
```

Comment:
```suggestion
monikerRange: '>= aspnetcore-5.0'
```

Reply:
W3CLogger is not available in 5.x, only 6.x